src/Contract/Transaction.purs

```diff
@@ -182,12 +184,14 @@ submit = wrapContract <<< map (wrap <<< unwrap) <<< QueryM.submitTxOgmios
 calculateMinFee
   :: forall (r :: Row Type)
    . Transaction
-  -> Contract r (Either ExportQueryM.ClientError Coin)
-calculateMinFee = wrapContract <<< QueryM.calculateMinFee
+  -> Contract r (Either ExportQueryM.ClientError Value)
+calculateMinFee = (map <<< map) (unwrap <<< toPlutusType <<< coinToValue)
+  <<< wrapContract
+  <<< QueryM.calculateMinFee
 
 -- | Same as `calculateMinFee` hushing the error.
 calculateMinFeeM
-  :: forall (r :: Row Type). Transaction -> Contract r (Maybe Coin)
+  :: forall (r :: Row Type). Transaction -> Contract r (Maybe Value)
```

Comment:
Hmm, I wonder if we should have an [Ada](https://playground.plutus.iohkdev.io/doc/haddock/plutus-ledger-api/html/Plutus-V1-Ledger-Ada.html#t:Ada) type for this. @ngua?